Src/Metrics/MetricTags.cs

```diff
@@ -9,50 +8,41 @@ namespace Metrics
     /// </summary>
     public struct MetricTags : Utils.IHideObjectMembers
     {
-        private static readonly string[] empty = new string[0];
+        private static readonly KeyValuePair<string,string>[] empty = Enumerable.Empty<KeyValuePair<string, string>>().ToArray();
```

Comment:
Any reason for using Enumerable.Empty instead of new KeyValuePair<string, string>[0]